.editorconfig

```diff
@@ -0,0 +1,34 @@
+# EditorConfig is awesome: https://EditorConfig.org
```

Comment:
Having a standard editor config is a nice idea.  I think this would be best handled as a separate issue/PR.  How about creating an issue and copying this into a code block in the issue as a starting point?

Reply:
Will do.

Reply:
See #117